cocos/core/data/decorators/property.ts

```diff
@@ -32,7 +32,7 @@ import { ClassStash, PropertyStash, PropertyStashInternalFlag } from '../class-s
 import { getClassName, mixin } from '../../utils/js-typed';
 
 // eslint-disable-next-line @typescript-eslint/ban-types
-export type SimplePropertyType = Function | string | typeof CCString | typeof CCInteger | typeof CCFloat | typeof CCBoolean;
+export type SimplePropertyType = Function | string | typeof CCString | typeof CCInteger | typeof CCBoolean;
```

Comment:
Why remove CCFloat?

Reply:
because both `CCInteger ` and `CCInteger ` are typeof `PrimitiveType`

Here is the type repeated

Reply:
It is `CCFloat`, not `CCInteger`.

Reply:
yes, my expression was wrong

what i wanna say is both `CCInteger`  and `CCFloat `  are typeof `PrimitiveType`